force-app/main/default/classes/CheckFulfillmentOrders_Batch.cls

```diff
@@ -0,0 +1,39 @@
+public class CheckFulfillmentOrders_Batch Implements Database.Batchable<sObject>, Database.Stateful {
+
+    public Database.QueryLocator start(Database.BatchableContext bc) {
```

Comment:
If you don't need to change the query dinamically and can use SOQL directly, please use the start method overload that returns a List so you can change its return type for List<FulfillmentOrder>